server.py

```diff
@@ -156,7 +156,9 @@ def update_recommendations(id):
 def delete_recommendations(id):
     if get_recommendations(id).status_code == 200:
         conn.execute("DELETE FROM recommendations WHERE id=%d" % id)
-    return '', HTTP_204_NO_CONTENT
+        return '', HTTP_204_NO_CONTENT
```

Comment:
@NicholasHyland can you please see our comments and push an update when you can?